dfastmi/io/ConfigFileOperations.py

```diff
@@ -263,9 +263,12 @@ def _update_to_absolute_path(
             Configuration for the D-FAST Morphological Impact analysis with only absolute paths.
         """
         relative_path = config.get(section, key, fallback="")
-        relative_path_converted_to_absolute_path = str(
-            Path(rootdir).joinpath(Path(relative_path))
-        )
+        if len(relative_path) > 0:
+            relative_path_converted_to_absolute_path = str(
+                Path(rootdir).joinpath(Path(relative_path))
+            )
+        else:
+            relative_path_converted_to_absolute_path = ""
```

Comment:
You can use `relative_path_converted_to_absolute_path = relative_path`
Since relative_path is already set to its fallback value of "" when the else is walked through.

Reply:
see PR #106

Reply:
Discussed with Bert.